From 139292ed87c6730454afc74e10a48643f3ec60ea Mon Sep 17 00:00:00 2001
:
: Upstreaming only makes sense once QEMU has proper support for this.
:
From: Simon Gaiser <simon@invisiblethingslab.com>
Date: Mon, 12 Feb 2024 11:18:40 +0100
Subject: [PATCH] libxl_pci: Pass power_mgmt via QMP

Signed-off-by: Simon Gaiser <simon@invisiblethingslab.com>
---
 tools/libs/light/libxl_pci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/libs/light/libxl_pci.c b/tools/libs/light/libxl_pci.c
index e87c58100127..a6b0e3adcb68 100644
--- a/tools/libs/light/libxl_pci.c
+++ b/tools/libs/light/libxl_pci.c
@@ -1243,6 +1243,8 @@ static void pci_add_qmp_device_add(libxl__egc *egc, pci_add_state *pas)
      */
     if (pci->permissive)
         libxl__qmp_param_add_bool(gc, &args, "permissive", true);
+    if (pci->power_mgmt)
+        libxl__qmp_param_add_bool(gc, &args, "power_mgmt", true);
 
     qmp->ao = pas->aodev->ao;
     qmp->domid = domid;
-- 
2.44.0

